index.html

```diff
@@ -65,7 +65,8 @@
 
   <!-- Bottom bar / filter input -->
   <div class="bottom-wrapper hidden">
-    <img src="app/assets/shuffle.svg" class="filter-icon">
+    <img id="filterIcon" src="app/assets/question.svg" class="filter-icon">
```

Comment:
let's use classes only, since tabs will be implemented soon (and there will be multiple filter-icons)